singer_sdk/plugin_base.py

```diff
@@ -155,6 +156,9 @@ def __init__(
         metrics._setup_logging(self.config)
         self.metrics_logger = metrics.get_metrics_logger()
 
+        # Initialization timestamp
+        self.__initialized_at = int(time.time() * 1000)
```

Comment:
@pnadolny13 I used an integer here since that feels easier to filter and dedup with in most targets but let me know if you think otherwise

Reply:
@edgarrmondragon yeah that makes sense to me!